contracts/Vault.vy

```diff
@@ -1637,6 +1645,13 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
         self.erc20_safe_transferFrom(self.token.address, msg.sender, self, totalAvail - credit)
     # else, don't do anything because it is balanced
 
+    # Update cached value of delegated assets
+    #   (used to properly account for mgmt fee in `_assessFees`)
+    self.delegatedAssets -= self._strategy_delegatedAssets[msg.sender]
+    delegatedAssets: uint256 = Strategy(msg.sender).delegatedAssets()
```

Comment:
I like that this approach is backward compatible with old base strategies and we can reuse almost all of our previous code

Reply:
Assuming the strategists filled this value out correctly, which hopefully is a reasonable assumption